__tests__/app.test.js

```diff
@@ -490,21 +500,35 @@ describe("express app", () => {
       });
     });
   });
-  describe('DELETE /api/comments/:comment_id', () => {
-    it('status 204: and no content when valid comment_id', async () => {
-      const {body, status} = await request(app).delete("/api/comments/1")
+  describe("DELETE /api/comments/:comment_id", () => {
+    it("status 204: and no content when valid comment_id", async () => {
+      const { body, status } = await request(app).delete("/api/comments/1");
       expect(status).toBe(204);
       expect(body).toEqual({});
     });
-    it('status 400: invalid comment ', async () => {
+    it("status 400: invalid comment ", async () => {
       const errBody = ERR_MSGS.PG["22P02"];
-      const {body} = await request(app).delete("/api/comments/1; SELECT * FROM users;")
+      const { body } = await request(app).delete(
+        "/api/comments/1; SELECT * FROM users;"
+      );
       expect(body).toEqual(errBody);
     });
-    it('status 404: comment does not exist', async  () => {
+    it("status 404: comment does not exist", async () => {
       const errBody = ERR_MSGS.DOES_NOT_EXIST;
-      const {body} = await request(app).delete("/api/comments/9001");
+      const { body } = await request(app).delete("/api/comments/9001");
       expect(body).toEqual(errBody);
     });
   });
+  describe("GET /api", () => {
+    it("status 200: responds with array of endpoints", async () => {
+      const { body: {endpoints}} = await request(app).get("/api");
+      Object.keys(endpoints).forEach((endpoint)=>{
```

Comment:
Id prefer a more exact test check. If we import the endpoints.json we can check that the response matches exactly :)